core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+- name: Download and unpack Helm binary
+  unarchive: 
+    src: "{{ apache_epirepo_path }}/files/helm-{{ helm_version }}-linux-amd64.tar.gz"
+    dest: "/usr/local/bin"
+    extra_opts:
+      - --strip=1
+      - --wildcards
+      - '*/helm'
+    remote_src: yes
```

Comment:
I'd suggest using `true/false` instead of `yes/no` here, as `yes/no` is specific to ansible and openssh, but `true/false` you can find everywhere.